src/react-components/ui-root.js

```diff
@@ -1110,45 +1122,16 @@ class UIRoot extends Component {
     const audioSetupPanel = this.state.entryStep === ENTRY_STEPS.audio_setup && this.renderAudioSetupPanel();
 
     // Dialog is empty if coll
-    let dialogContents = null;
-
-    if (this.state.entryPanelCollapsed && !this.isWaitingForAutoExit()) {
-      dialogContents = (
-        <div className={entryStyles.entryDialog}>
-          <div>&nbsp;</div>
-          <WithHoverSound>
-            <button onClick={() => this.setState({ entryPanelCollapsed: false })} className={entryStyles.expand}>
-              <i>
-                <FontAwesomeIcon icon={faChevronUp} />
-              </i>
-            </button>
-          </WithHoverSound>
-        </div>
```

Comment:
I guess you know this already, but one downside to removing collapse is that the view is almost entirely blocked in landscape on iPhone 5/SE and you can't tap the user list or help buttons

![image](https://user-images.githubusercontent.com/79419/50669593-e68df300-0f7a-11e9-9dfa-1408a3e482e3.png)

Reply:
yeah -- the problem is that the collapse button can fool the user into thinking they've entered the room -- it's actually worse on the devices where the panel takes up the most space because it feels like you've "clicked through" the dialog

Reply:
👍 makes sense

Reply:
I thought about this more and i think the right solution here is to add a grip point at the top the user can use to slide the panel up and down -- we can do that at some point